installer/targets/ros/install.bash

```diff
@@ -31,7 +33,7 @@ TUE_DEV_DIR=$TUE_ENV_DIR/dev
 if [ ! -f $TUE_SYSTEM_DIR/devel/setup.bash ]
 then
     mkdir -p $TUE_SYSTEM_DIR/src
-    hash g++ 2> /dev/null || sudo apt-get install --assume-yes g++
+    hash g++ 2> /dev/null || tue-install-system-now g++
```

Comment:
@Timple We don't need the hash check here anymore, right? But it is faster than the dpkg check right? Than it is usefull to keep these.

Reply:
I don't mind either way. tue-install-system-now is more descriptive and
consistent.

Hash is 0.07s faster on my PC.